src/api/chats/index.ts

```diff
@@ -0,0 +1,18 @@
+import {Router} from 'express';
+import {validationMiddleware} from '../../tools/wrapper.helpers';
+import {getChatMessages, getMyChats} from "./get";
+import {checkEntityId} from '../../tools/wrapper.helpers';
+import {ChatEntity} from "../../db/entities/chat.entity";
+import {MessageEntity} from "../../db/entities/message.entity";
+import {deleteMessage} from "./delete";
+import {createMessage} from "./post";
+import {PostMessageRequest} from './requests/post-message.request';
+
+const router = Router();
+
+router.get('/', getMyChats);
+router.get('/:id/messages', checkEntityId(ChatEntity), getChatMessages);
+router.post('/:id/messages', checkEntityId(ChatEntity), validationMiddleware(PostMessageRequest), createMessage);
+router.delete('/:chatId/messages/:id', checkEntityId(MessageEntity), deleteMessage);
```

Comment:
Why do u need `chatId` param in this endpoint? It seems like that param don't use anywhere.